posthog/queries/app_metrics/test/test_app_metrics.py

```diff
@@ -0,0 +1,239 @@
+from typing import Optional
+
+from freezegun.api import freeze_time
+
+from posthog.kafka_client.client import ClickhouseProducer
+from posthog.kafka_client.topics import KAFKA_APP_METRICS
+from posthog.models.app_metrics.sql import INSERT_APP_METRICS_SQL
+from posthog.models.event.util import format_clickhouse_timestamp
+from posthog.queries.app_metrics.app_metrics import AppMetricsQuery
+from posthog.queries.app_metrics.serializers import AppMetricsRequestSerializer
+from posthog.test.base import BaseTest, ClickhouseTestMixin, snapshot_clickhouse_queries
+from posthog.utils import cast_timestamp_or_now
+
+
+def create_app_metric(
+    team_id: int,
+    timestamp: str,
+    plugin_config_id: int,
+    category: str,
+    job_id: Optional[str] = None,
+    successes=0,
+    successes_on_retry=0,
+    failures=0,
+):
+    timestamp = cast_timestamp_or_now(timestamp)
+    data = {
+        "timestamp": format_clickhouse_timestamp(timestamp),
+        "team_id": team_id,
+        "plugin_config_id": plugin_config_id,
+        "category": category,
+        "job_id": job_id or "",
+        "successes": successes,
+        "successes_on_retry": successes_on_retry,
+        "failures": failures,
+    }
+    p = ClickhouseProducer()
+    p.produce(topic=KAFKA_APP_METRICS, sql=INSERT_APP_METRICS_SQL, data=data)
```

Comment:
don't we need to wait for this to be ingested in tests to avoid flakiness?

Reply:
No - test data is ingested instantly in python tests.